_includes/binary2.html

```diff
@@ -23,7 +23,7 @@
         <div class="opcol">AND</div>
         <div class="opcol">AND</div>
         <div class="opcol">AND</div>
-    </div><input type="button" value="Calculate" onClick="set_bits();" class="ok" style="margin-top: 22px"><br style="clear: left">
+    </div>
```

Comment:
@CodeSarthak I think you have added an extra closing div tag. Please check and update your PR.

Reply:
You can share a screenshot after removing  button also.